app/javascript/packages/submit-button/submit-button-element.spec.ts

```diff
@@ -1,3 +1,4 @@
+import { mock } from 'node:test';
```

Comment:
Using `node:test` `mock` here instead of `sinon` because I think it might be possible to eventually move toward dropping Sinon in the future, this making that future work simpler.

Continuing to use Sinon in `spinner-button-element.spec.ts` for consistency's sake, since it's already used there.